duckduckgo/Android
# Feature/basic web browsing

Asana: https://app.asana.com/0/471012189846020/471193197990030

**Description**:
Basic implementation of a `WebView` with functioning `back/forward/refresh` buttons and the omnibar.

The UI is bare as designs are not finalised.

The omnibar entry differentiates between search queries and URLs. In both cases, it adds the required `t` and `tappv` query parameters.

**Steps to test this PR**:
1. Search for a simple query - "horse"
    - Ensure it performs a DDG search
2. Search for a more complex search - "4 + 5"
    - Ensure it performs the DDG search
3. Enter a URL - "https://example.com"
    - Ensure it loads the URL (and does not perform a search for it)
4. Repeat for more _exotic_ URL syntaxes


###### Author Checklist:
- [x] Ensure that there is a testing strategy (and document any non-automated tests)
- [x] Consider implications (CPU, memory, database / preference migrations, statistics)


###### Reviewer Checklist:
